### PR TITLE
Add project Bergamot

### PR DESCRIPTION
**Pull request by djmitche**
Per email from Kelly Davis

### CODE REVIEW

**Comment by djmitche** [comment]
@kdavis-mozilla how does this look?  This will give you a dedicated worker pool, `project-bergamot/ci`, running docker-worker in CI.  There's one TODO in there, too.  I can list you as admin, but maybe there's already a GH team in place we could use?

You'll also need to ask an admin of the @mozilla GH org to add the `marian-dev` repository to the [GitHub App](https://github.com/apps/community-tc-integration).

File: config/projects/bergamot.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+bergamot:
+  adminRoles: [] # TODO github teams or users who should have admin access to this project
```
Comment (kdavis-mozilla) [inline config/projects/bergamot.yml]: I just created a team for this [github-team:mozilla/bergamot](https://github.com/orgs/mozilla/teams/bergamot) which right now only has myself.

File: config/projects/bergamot.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  repos:
+    - github.com/mozilla/marian-dev:*
+  workerPools:
```
Comment (kdavis-mozilla) [inline config/projects/bergamot.yml]: I'd need linux, windows, and mac workers as I need to make sure the code continues to work there too.

File: config/projects/bergamot.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    ci:
+      owner: nobody@mozilla.org
+      emailOnError: false
+      imageset: docker-worker
+      cloud: gcp
+      minCapacity: 0
+      maxCapacity: 10
```
Comment (kdavis-mozilla) [inline config/projects/bergamot.yml]: Would I need clients for each of the various worker types, e.g. a windows client?

File: config/projects/bergamot.yml (hunk 2 above)
Comment (djmitche) [reply config/projects/bergamot.yml]: We can add a Windows cloud-based worker-pool, too.  However, nobody offers macs in pools, so for that platform we'll need to find a mac (or more than one).  That might be in MacStadium, or another mac-rental company, or a mac in a Mozilla office somewhere.  That might be a good next step after we get the rest started.

File: config/projects/bergamot.yml (hunk 1 above)
Comment (djmitche) [reply config/projects/bergamot.yml]: Sounds good!

File: config/projects/bergamot.yml (hunk 3 above)
Comment (djmitche) [reply config/projects/bergamot.yml]: For the most part, you just create a worker pool for each.  For non-cloud things (like macs), currently that requires creating a client, but we're working on some final blockers to making that easy, too.  So we'll see how the timing works out.

**Comment by djmitche** [comment]
Updated.  The worker pool name changed, to include context.  It's now `project-bergamot/ci-linux` and `project-bergamot/ci-windows`.

File: config/projects/bergamot.yml (hunk 2 above)
Comment (kdavis-mozilla) [reply config/projects/bergamot.yml]: Thanks on the windows workers.

And on the Mac workers, yeah the Deep Speech mac sits one the desk next to me. I hear every PR :)

**Comment by djmitche** [comment]
applied (I think.. lots of grepping around going on)

**Comment by kdavis-mozilla** [comment]
@djmitche One question, which may be stupid as I'm learning TC...

[bergamot.yml](https://github.com/mozilla/community-tc-config/blob/master/config/projects/bergamot.yml) specifies a ci-linux worker pool. But looking at the [page](https://community-tc.services.mozilla.com/provisioners/proj-bergamot) showing workers for proj-bergamot there are no ci-linux workers only ci-windows workers. Is this correct?

**Comment by djmitche** [comment]
It's not a stuipd question -- the UI is misleading :)  The worker page you're looking at shows only workers that have actually run.  So I'm guessing that there haven't been any ci-linux workers lately, and thus the worker pool does not appear there.  Both worker pools do appear [here](https://community-tc.services.mozilla.com/worker-manager).  There's work currently underway to make this better!

**Comment by kdavis-mozilla** [comment]
Thanks!

**Pull request merged**